README.md

```diff
@@ -2,3 +2,16 @@
 [![Build Status](https://travis-ci.org/SMaxOwok/ui-pictionary.svg?branch=master)](https://travis-ci.org/SMaxOwok/ui-pictionary)
 
 ![picationary](static/pictionary.gif)
+
+# Setup
+Requirements: Ruby version 2.5.7, Bundler 2
+- Install nvm and yarn
```

Comment:
I'm partial towards pointing people to `nodenv` instead of `nvm`.  Same with ruby, if you want to add a little more about those.